test/integration/targets/dcnm_vrf/tests/nxapi/create.yaml

```diff
@@ -0,0 +1,68 @@
+---
```

Comment:
Any reason to not put these three tests in the same test file?

Reply:
Just to keep operations separate, got inspired by some existing tests where I saw different files for different operations. Is there a style guideline for int tests?

Reply:
The separate tests style predates my time with the team. When I came on most tests were combined in a single `sanity` test. But really it's whatever makes sense, which might be separate test files, especially if a given test operation has a lot of complicated tests.

Note there's also an overall CI duration cost for each separate test so if they can be combined that's usually preferred. For `network` I'm planning to combine them all in one file.

Reply:
All right, will move the tests to one file, sanity.yaml